scripts/all-tests.sh

```diff
@@ -98,14 +98,12 @@ END
     "unit")
         case "$SHARD" in
             "lms")
-                paver test_system -s lms --extra_args="--with-flaky" || { EXIT=1; }
-                paver coverage
+                SHARD=1 paver test_system -s lms --extra_args="--with-flaky" --cov_args="-p" || { EXIT=1; }
```

Comment:
Oh, I see what's confusing now. It's that we are using TEST_SUITE = unit. Would it make sense to have a new flavor of TEST_SUITE here? Like 'lms-unit', which then has SHARDs that are numeric? This would make the translation between unit shards and bok-choy shards more even. 

[That said, perhaps this would be part of the "actual new shard" PR, as opposed to "enable ability to shard" PR, which is this one. Thoughts?]